src/main/java/com/android/volley/toolbox/JsonObjectRequest.java

```diff
@@ -35,44 +37,120 @@
     /**
      * Creates a new request.
      *
-     * @param method the HTTP method to use
      * @param url URL to fetch the JSON from
-     * @param jsonRequest A {@link JSONObject} to post with the request. Null indicates no
-     *     parameters will be posted along with request.
      * @param listener Listener to receive the JSON response
      * @param errorListener Error listener, or null to ignore errors.
      */
     public JsonObjectRequest(
-            int method,
-            String url,
-            @Nullable JSONObject jsonRequest,
-            Listener<JSONObject> listener,
-            @Nullable ErrorListener errorListener) {
+        String url,
+        Listener<JSONObject> listener,
+        @Nullable ErrorListener errorListener
+    ) {
         super(
-                method,
-                url,
-                (jsonRequest == null) ? null : jsonRequest.toString(),
-                listener,
-                errorListener);
+            Method.GET,
+            url,
+            null,
+            listener,
+            errorListener
+        );
     }
 
     /**
      * Constructor which defaults to <code>GET</code> if <code>jsonRequest</code> is <code>null
      * </code> , <code>POST</code> otherwise.
      *
-     * @see #JsonObjectRequest(int, String, JSONObject, Listener, ErrorListener)
+     * @deprecated Use {@link #JsonObjectRequest(int, String, JSONObject, Listener, ErrorListener)}.
      */
+    @Deprecated
```

Comment:
Marking this deprecated in consistency with JsonRequest constructor that is marked deprecated due to auto-selecting the method.